cartesia/_async_websocket.py

```diff
@@ -105,6 +108,95 @@ async def no_more_inputs(self) -> None:
             context_id=self._context_id,
             continue_=False,
         )
+    
+    async def flush(self) -> Callable[[], AsyncGenerator[Dict[str, Any], None]]:
+        """Trigger a manual flush for the current context's generation. This method returns a generation that yields the audio prior to the flush."""
+        await self.send(
+            model_id=DEFAULT_MODEL_ID,
+            transcript="",
+            output_format=TTS.get_output_format("raw_pcm_f32le_44100"),
```

Comment:
should we define the default output format somewhere instead of passing to `self.send` each time as a 'magic constant'

Reply:
this makes almost too much sense